### DISCUSSION TITLE
URL request in aliases

### DISCUSSION THREAD

**Discussion by FrancyMak** [Zrips/CMI#7434] [Enhancement]
Hi, I would very useful like it if the ability to send requests to a given url in aliases was added, because it is a current feature of MyCommand (the only one that still makes me use that plugin in the server).

**Comment by CloudeLecaw** [comment]
what you mean by requests to an url?
normaly all that can be done is print an url to chat and the request winodws options if you click on them.
MyCommand is not one of the more known plugins like WorldEdit...  So could you descripe a bit deeper what this plugins excalcy was/is doing?
This would help to take a bit deeper look into what you are requesting :)

**Reply by FrancyMak** [reply]
Sure!

![image](https://user-images.githubusercontent.com/75610581/207804470-c08d66e3-d582-41c1-a8be-ecff71c5460c.jpeg)

Here you can see the wiki, I think that will explain the feature better than me.
